Src/IO.c

```diff
@@ -192,7 +192,7 @@ void detectInput(){
 //	if ((smallestnumber > 2000 )&&(smallestnumber < 3000)){
 //		oneshot42 = 1;
 //	}
-		if (smallestnumber > 3000){
+		if (smallestnumber > 90){		//Was 3000, lowered for compatability with spektrum surface recievers
```

Comment:
This can't be like this, oneshot and multishot will also full in this case

Reply:
What's the maximum expected value for oneshot and multishot? I can test out 10 above that to see if it works.
Since they already appeared disabled, I just chose a value 10 above what is enabled.